### ISSUE TITLE
Result on validation set

### ISSUE THREAD

**Issue by yzhou-saic** [kujason/avod#7]
1) I check the training split for validation, that's the same with the one released by MV3D paper, is that the case ?
2) By the split, on **validation set**, I run the "avod_cars_example.config", and get results after 120,000 iterations, in "avod_cars_example_results_05_iou_0.1.txt"
    "car_detection_3D_AP: 89.85737     80.741768        80.542282"
   Is it reasonable results ?
3) Checked the MV3D paper, it seems they report better results on the same splitted validation set ?

**Comment by kujason** [comment]
1. Yes, the training and validation splits are the same splits that are used by many others including MV3D, and are used for fair comparison with other methods.
2. The final result at step 120,000 may not provide the best results. We recommend running `scripts/offline_eval/plot_ap.py` to find the best performing checkpoint at moderate difficulty. Checkpoints are evaluated at both 0.7 and 0.5 IoU for cars, while they are evaluated at both 0.5 and 0.25 IoU for pedestrians and cyclists.
3. At 0.5 3D IoU, we find our network achieves around 88-89% AP for cars, similar to that of MV3D. However, car detections at 0.5 3D IoU may not be localized very well, and the 0.7 3D IoU is used as the benchmark for comparison, which is consistent with the KITTI test server benchmark.

**Comment by yzhou-saic** [comment]
All the above questions are under the context of validation (testing not done by me yet):

For 2, I ran checkpoints from 10,000 to 120,000, every 10,000 iterations, basically the 12 checkpoints achieve similar around "89.85737 80.741768 80.542282", is that possible there is a much better one in the middle ? 

For 3, on the **_validation set_**, it seems "MV3D" achieved "96.02  89.05   88.38" for AP_3D@IoU=0.5, do you find that similar to my report ? I completely understand validation set does not mean good generalization, and the algorithm can generalize well on the testing, but it may be quite helpful for verification.

**Comment by kujason** [comment]
Test set labels are not provided by KITTI, so other than the reported results from the test server, our comparisons are only on the provided validation split.

What were your results for cars at 0.7 3D IoU? Your results at 0.5 3D IoU do not seem typical and we would recommend retraining the network, leaving the configurations as the defaults, where checkpoints are saved and evaluated every 1000 steps.

As I mentioned earlier, we do not use 0.5 3D IoU for comparison with car results. We simply left the low (0.5) IoU evaluation in the code for reference.

**Comment by yzhou-saic** [comment]
Thanks for the clarification, and many thanks for the code release.
